content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -0,0 +1,154 @@
+---
+reviewers:
+- klueska
+- pohly
+title: Dynamic Resource Allocation
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+Dynamic resource allocation is a new API for requesting and sharing resources
+between pods and containers inside a pod. Third-party resource drivers are
+responsible for tracking and allocating resources. The different kinds of
+resources support arbitrary parameters for defining requirements and
+initialization.
+
+## {{% heading "prerequisites" %}}
+
+Kubernetes v{{< skew currentVersion >}} includes cluster-level API support for
+dynamic resource allocation, but it [needs to be
+enabled](#enabling-dynamic-resource-allocation) explicitly.  You also must
+install a resource driver for specific resources that are meant to be managed
+using this API.  If you are not running Kubernetes v{{< skew currentVersion>}},
+check the documentation for that version of Kubernetes.
+
+<!-- body -->
+
+## API
+
+The new `resource.k8s.io/v1alpha1` {{< glossary_tooltip text="API group"
+term_id="api-group" >}} provides four new types:
+
+- ResourceClass: defines which resource driver handles a certain kind of
```

Comment:
Ideally these should link to the API documentation (similar to `/docs/reference/kubernetes-api/config-and-storage-resources/csi-storage-capacity-v1/`. But I don't know yet whether the new API group will have such generated documentation and where it will land.

Do we need to configure that?

Reply:
Maybe for now, link each of them to the section in the KEP that discusses them? Then we can update the links once they are they are available as above).

Reply:
There's no good anchor in the KEP for these. They would all link to the "API" section. I prefer to keep this as is until we have some generated API documentation.